app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -96,6 +97,10 @@ export default {
           return
         }
 
+        if (this.isClanCreator()) {
+          this.fetchStudentNamesForTeacher(me.id)
+        }
```

Comment:
This implies the clan creator is always a teacher. However we can have situations where the clan creator isn't a teacher leading to issues here.

For example for a third party, we may create a principle account and make them a clan owner of their team. Their account will not have classrooms or students, and needs to instead get the student names from just being a clan creator.